app/services/stripe_service/report.rb

```diff
@@ -127,6 +253,33 @@ def create_payout
     }
   end
 
+  def create_intent
+    {
+      "stripe_op": "create_intent",
+      "transaction_id": tx.id,
+      "stripe_seller_id": stripe_account.stripe_seller_id
+    }
+  end
+
+  def cancel_intent
+    {
+      "stripe_op": "cancel_intent",
+      "transaction_id": tx.id,
+      "stripe_seller_id": stripe_account.stripe_seller_id
```

Comment:
Minor thing: we could report the `stripe_payment_intent_id` here. We have the ID available both when starting the call and when it completes.